### PR TITLE
Development

### PR DESCRIPTION
**Pull request by n-maido**
Backend:
- Integrated browse events page #7 
- User associated with their events that they are hosting
- Added graphql queries for allEvents, allUsers, eventById, usersById
- Added graphql mutations for createEvent, createUser

Frontend:
- Implemented landing page #18 
- Refactored browse_events into display_events, which is now reusable
- Implemented individual event page #8 
- Event page button is now reusable and changes styling and action depending on current page
- Completed routing
- Add popup notifications for delete event and cancel registration
- Integrated event page with backend #22 
- Integrate create event form with backend #26 
- Integrate create account form with backend #25 
- Implemented Login and event registration forms
- Added transitionsPage to swipe to previous page. commit 13698c17bc1cb196f920ceb2c671b86569a3bc89

Screen Shot 2021-08-13 at 9 29 15 AM
Screen Shot 2021-08-10 at 4 09 16 PM
Screen Shot 2021-08-13 at 9 30 40 AM
Screen Shot 2021-08-13 at 9 31 04 AM

### CODE REVIEW

**Comment by n-maido** [comment]
@YenyuHuang @manlalaro1 @shivichaubey

File: backend/auth/mutations.py [hunk 1]
```diff
@@ -14,5 +14,31 @@ def mutate(self, parent, input=None):
...
+#     name: String!
+#     tag: String!
+#     organizer: User!
+#     location: String!
+#     description: String!
+
+# type User:
+#     id: ID!
+#     username: String!
+#     password: String!
```
Comment (yashika51) [inline backend/auth/mutations.py:44]: No commented code please.

File: frontend/lib/main.dart [hunk 2]
```diff
@@ -159,24 +212,39 @@ class _LoginPageState extends State<LoginPage> {
...
+            //   onPressed: () {
+            //     Route route = MaterialPageRoute(
+            //         builder: (context) =>
+            //             CreateAccount(account: this.account, isAdd: false, screen: null,));
+            //     Navigator.push(context, route);
+            //   },
+            // style: TextButton.styleFrom(primary: Palette.highlight_1)
+
+            // ),
           ],
```
Comment (yashika51) [inline frontend/lib/main.dart:248]: Same as above.

File: frontend/lib/main.dart [hunk 3]
```diff
@@ -203,29 +272,41 @@ class _LoginPageState extends State<LoginPage> {
...
+  //           addMutation.addAccount(id, firstName, lastName, email, password)),
+  //     ),
+  //   );
+  //   if (result.data["success"] != null) {
+  //     print('The user was created successfully!');
+  //   } else {
+  //     print('There was an error!');
+  //     print(result.data["register"]["errors"]["email"][0]["message"]);
+  //   }
+  // }
```
Comment (yashika51) [inline frontend/lib/main.dart:310]: Same as above.

**Review by yashika51 — COMMENTED** [review]
Hi Guys, I think in general Frontend has a lot of commented code that should be removed. Other than that I see a lot of yet to implemented comments which is fine, just make sure to work on that before moving forward. 
I don't think there are any code specific changes needed until you complete the functionalities. 
Good to see use of `.env` files and improvements from last week's review. 
I'd like to emphasise again on documenting your code

Major problem carried forward from last week as well is I don't see even commits for all the teammates.

**Pull request merged**